scripts/crate/interface.mts

```diff
@@ -0,0 +1,38 @@
+import { workingDirectory } from '../helpers/utils.mjs';
+import { getDefaultCrateEnvironment, type CrateEnvironment } from './index.mts';
+
+export function getInterfaceCrateEnvironment(): CrateEnvironment {
+  return {
+    ...getDefaultCrateEnvironment(),
+    cratePath: path.join(workingDirectory, 'interface'),
+    manifestPath: path.join(workingDirectory, 'interface', 'Cargo.toml'),
+    getLintClippyArguments,
+    getLintDocsArguments,
+    getLintFeaturesArguments,
+  };
+}
+
+function getLintClippyArguments(cliArguments: string[]): string[] {
+  const args = ['--all-targets', '--all-features', ...cliArguments];
+  // Check whether a '--' was already used.
+  if (args.indexOf('--') === -1) {
+    args.push('--');
+  }
+  // Add additional arguments.
+  args.push(
+    '--deny=warnings',
+    '--deny=clippy::default_trait_access',
+    '--deny=clippy::arithmetic_side_effects',
+    '--deny=clippy::manual_let_else',
+    '--deny=clippy::used_underscore_binding'
+  );
+  return args;
+}
+
+function getLintDocsArguments(cliArguments: string[]): string[] {
+  return cliArguments; // TODO
+}
```

Comment:
TODO?

Reply:
This file was removed as it is not needed anymore.